draft-ietf-scitt-architecture.md

```diff
@@ -877,11 +877,23 @@ Such policies may use as input all information in the Envelope, the Receipt, and
 
 # Privacy Considerations
 
-Transparency Services MAY support anonymous access.
-Issuers SHOULD ensure Signed Statements submitted to public access services are acceptable for public disclosure.
-Publicly accessible Signed Statements MUST NOT carry confidential information.
-Once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
-In some deployments, a special role, such as an Auditor, might require access to Signed Statements.
+All contents exchanged between actors is protected using secure authenticated channels (e.g., TLS) but this does not necessarily exclude network traffic analysis.
+
+The Transparency Service is trusted with the confidentiality of the Signed Statements presented for Registration. Issuers are responsible for verifying that the privacy and security posture of the Transparency Service is suitable for the contents of their Signed Statements prior to Registration. In particular, Issuers must carefully review the inclusion of private, confidential, or personally identifiable information (PII) in their Statements against the privacy posture of the Transparency Service.
+This document supports opaque references (such as hashes) for cases where information needs to be included on the log whilst remaining secret.
```

Comment:
```suggestion
Using a hash as the statement about an artifact, supports cases where information needs to be included on the append-only log whilst remaining opaque to the transparency service.
```

Reply:
Not a reference to Hash Envelope specifically, just pointing out that people can commit to data with disclosing it.

I like your suggestion and will commit it.